### PR TITLE
Add section for user data field formatting

### PR DESCRIPTION
**Pull request by bobbyatsegment**
### Proposed changes

Facebook CAPI (Actions) docs do not currently provide insight into what formatting the integration performs to user data values prior to hashing them. Added this information as a section under FAQ.

### CODE REVIEW

File: src/connections/destinations/catalog/actions-facebook-conversions-api/index.md [hunk 1]
```diff
@@ -149,6 +149,19 @@ Segment creates a SHA-256 hash of the following fields before sending to Faceboo
 
 If you use Facebook Pixel, the Pixel library also hashes the External ID. This means External IDs will match across Facebook Pixel and Facebook Conversions API if they use the External ID for [deduplication](https://developers.facebook.com/docs/marketing-api/conversions-api/deduplicate-pixel-and-server-events/#fbp-or-external-id){:target="_blank"}.
 
+### User Data Formatting
+
+Segment applies formatting to User Data Parameters as follows:
+
+| User Data Field                                                                     | Formatting applied to field value before hashing                                                                                                               |
+|-------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| External ID, Email, Gender,  Last Name, First Name,  City, State, Zip Code, Country | All whitespace is removed from string, set to lowercase.                                                                                                       |
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-facebook-conversions-api/index.md:158]: Do these "stack"? I see the State field in this row, which gets converted to lowercase here, and then in line 162 it looks like it's compared against a map object of states and their two-character ANSI code. 

Would it be formatted like this: 
`Texas` > `texas` > `tx`?

Comment (forstisabella) [inline src/connections/destinations/catalog/actions-facebook-conversions-api/index.md:158]: I recommend making each of these user data fields in this line their own line in the table. If these do "stack" (Texas > texas > tx) you could combine the two transformations in one line. For example: 

"All whitespace is removed from string and the result is compared against a map object of states and their two-character ANSI abbreviation code. Example: "Texas", "TX", or "tx" in this field will be formatted as "tx"."

File: src/connections/destinations/catalog/actions-facebook-conversions-api/index.md [hunk 2]
```diff
@@ -149,6 +149,19 @@ Segment creates a SHA-256 hash of the following fields before sending to Faceboo
...
+| Phone                                                                               | All non-numeric characters are removed from string.                                                                                                            |
+| Gender                                                                              | A string of 'male' will be transformed to 'm', and 'female' to 'f'.                                                                                            |
+| Date of Birth                                                                       | No formatting is applied.                                                                                                                                      |
+| State                                                                               | Compared against a Map object of states and their 2-character ANSI abbreviation code. Example: "Texas", "TX", or "tx" in this field will be formatted as "tx". |
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-facebook-conversions-api/index.md:162]: We spell out one to ten for readability: 11+ can be represented with numbers!
```suggestion
| State                                                                               | Compared against a Map object of states and their two-character ANSI abbreviation code. Example: "Texas", "TX", or "tx" in this field will be formatted as "tx". |
```

File: src/connections/destinations/catalog/actions-facebook-conversions-api/index.md [hunk 3]
```diff
@@ -149,6 +149,19 @@ Segment creates a SHA-256 hash of the following fields before sending to Faceboo
...
+| Country                                                                             | Compared against a Map object of countries and their 2-letter ISO 3166-1 alpha-2 country codes.                                                                |
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-facebook-conversions-api/index.md:163]: We spell out one to ten for readability: 11+ can be represented with numbers!
```suggestion
| Country                                                                             | Compared against a Map object of countries and their two-letter ISO 3166-1 alpha-2 country codes.                                                                |
```

**Comment by forstisabella** [comment]
Hi @bobbyatsegment - left a few comments on this PR, if you have a minute to address them!

File: src/connections/destinations/catalog/actions-facebook-conversions-api/index.md (hunk 1 above)
Comment (bobbyatsegment) [reply src/connections/destinations/catalog/actions-facebook-conversions-api/index.md:158]: Exactly right! first the state is lowercased, then replaced with the corresponding value in the map

Comment (bobbyatsegment) [reply src/connections/destinations/catalog/actions-facebook-conversions-api/index.md]: Love this, thank you! Just updated with this - separated each user data field in their own line, and used the wording you recommended here for both state and country fields

**Review by forstisabella — APPROVED** [review]

**Comment by forstisabella** [comment]
sweet, this looks great: I'll go ahead and merge it! @bobbyatsegment

**Pull request merged**